File: app/src/main/java/com/blog/demo/custom/widget/GroupOneView.java
package com.blog.demo.custom.widget;

import android.content.Context;
import android.util.AttributeSet;
import android.view.View;
import android.widget.LinearLayout;
import android.widget.TextView;

import androidx.annotation.Nullable;

import com.blog.demo.R;

public class GroupOneView extends LinearLayout {

    public GroupOneView(Context context) {
        super(context);
    }

    public GroupOneView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);

        View.inflate(context, R.layout.view_group_one, this);

        TextView tv =  findViewById(R.id.text_view);
        tv.setText("This is group one.");
    }

}

File: app/src/main/java/com/blog/demo/control/widget/GroupOneView.kt
package com.blog.demo.control.widget

import android.content.Context
import android.util.AttributeSet
import android.widget.LinearLayout
import android.widget.TextView
import com.blog.demo.R

class GroupOneView(context: Context, attrs: AttributeSet?, defStyleAttr: Int) :
        LinearLayout(context, attrs, defStyleAttr) {

    constructor(context: Context) : this(context, null)

    constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0)

    init {
        inflate(context, R.layout.view_group_one, this)

        val tv: TextView = findViewById(R.id.text_view)
        tv.text = "This is group one."
    }

}